jekyll/_cci2/server-3-operator-externalizing-services.adoc

```diff
@@ -0,0 +1,84 @@
+---
+version:
+- Server v3.x
+- Server Admin
+---
+= CircleCI Server v3.x Configuring External Services
+:page-layout: classic-docs
+:page-liquid:
+:icons: font
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== External Database and Vault Setup for CircleCI Installations
+This document describes how to configure the following external services for use with CircleCI Server 3.0 Installation:
+
+* PostgreSQL v12.6
+
+== PostgreSQL
+
+=== Best Practices for your PostgreSQL
+[width="60%",frame="topbot",options="header,footer"]
+Consider running at least two or more PostgreSQL replicas to enable recovery from primary failure and for backups. Following are the recommended specifications of the PostgreSQL machines:
+
+|=====================================================================================
```

Comment:
I will check this branch out and fix the table formatting, it's quite different in asciidoc :-)